### DISCUSSION TITLE
Return the next day: Wagmi says connected, requests donnot show up (metamask, wallet connect)

### DISCUSSION THREAD

**Discussion by Marcelh1983** [wevm/wagmi#2064] [Q&A]
Hi,

I'm building an application, where you can login and do transactions.
Everything seems to work fine if I connect my wallet using Metamask via Wallet Connect and after that execute some transactions.

If a return a few hours later, or the next day, Wagmi says my wallet is still connected, but when I execute a transaction, the requests are not popping up in my metamask wallet. 

What would be the problem here? Is there a way to determine this situation?

**Comment by tmm** [comment]
Can you drop a link to a minimal reproducible example? Happy to look into it.

**Reply by Marcelh1983** [reply]
Hi @tmm I think I found what caused the issue; I tried to create a simple reproducible example bit could reproduce the issue with only the walletConnect connector. Then I added more stuff just like in my application: metamask and coinbase connectors. Since I added these two the issue occurs. When I login and switch network it works fine. When I come back a few hours later, I doesnt work. Like I said, this seemed to work with just the walletConnect connector.

Here my [codesandbox](https://codesandbox.io/s/wagmi-connect-issue-zik14k?file=/src/App.tsx)

**Reply by Marcelh1983** [reply]
Now it works again...it's annoying and time consuming to test. I experience this issue in my application all the time when I come back a few hours later. Hope you can find the issue. Meanwhile I'll try some things to have a 100% reproducible example
